third_party/move/evm/move-to-yul/tests/AccountStateMachine.v2_exp

```diff
@@ -0,0 +1,1138 @@
+/* =======================================
```

Comment:
I think I said before we should keep the evm stuff alone. Why are we adding this additional complexity?

Reply:
It was easy enough to add it.  Just a line in a script.